### DISCUSSION TITLE
Welcome to Symfony Discussions!

### DISCUSSION THREAD

**Discussion by fabpot** [symfony/symfony#42363] [Announcements]
We are experimenting with using Github discussions as an additional way of getting support.

We are actively looking for community members to help support our users here. As long as we can demonstrate that there is enough questions and answers here to make this forum sustainable, we will keep this section active.

**Comment by vasilvestre** [comment]
> We are experimenting with using Github discussions as an additional way of getting support.
> 
> We are actively looking for community members to help support our users here. As long as we can demonstrate that there is enough questions and answers here to make this forum sustainable, we will keep this section active.

What can we discuss here? General Symfony topics, support for "own code" problems that use Symfony? Does it affect #support on Slack? Thank you for this initiative!

**Reply by fabpot** [reply]
I think the idea is to see if it can replace Slack as the main support channel.

**Reply by ruudk** [reply]
The issue with Slack that I have is that it only keeps track of the last 10.000 messages. That means that a lot of useful knowledge is lost every day. Hopefully by moving to Discussions we can persist that knowledge 😊

**Reply by vasilvestre** [reply]
I was hoping for a "new support" here, many questions already have been discussed thousands of times, that will be great I hope!

**Comment by linaori** [comment]
Sounds like a good idea to try!

I'll miss the direct engagement part and spinoff discussions, though that's nothing compared to the gain of discoverability for existing issues.

**Reply by arfaram** [reply]
Discussions or a persistent searchable history? IMHO I prefer the second one. When I'm using slack I try always to copie useful code snippet somewhere in my Notes, gist etc because they will later removed from slack. The idea here is to save discussions somehow. If the community moves slowly here then we will have kind of wiki,forum or own stackoverflow😉

**Comment by StaffNowa** [comment]
Good idea. I like it 😏👍

**Reply by nawafwb8-alt** [reply]
<img width="1280" height="720" alt="8f85dfe3-6c6a-40e9-82ec-273cc89ba75a" src="https://github.com/user-attachments/assets/ce4e51d2-f101-47b3-8b84-5818b8c6e7d6" />

**Comment by zairigimad** [comment]
Great idea :)

**Comment by ricardodevries** [comment]
What a great experiment. Let's hope it stays.

**Comment by ltdangle** [comment]
We need a proper forum like back with the symfony1 days.

**Comment by ro0NL** [comment]
comments like https://github.com/symfony/symfony/issues/32080#issuecomment-901097229 by @carsonbot  should be moved to discussions btw, not issues/PRs IMHO

i think we're better of closing unanswered (with actitvty) over time, rather than piling up

**Comment by authentictech** [comment]
I'd like to see a forum for new Symfony developers and/or a forum for people working through Symfony 5:The Fast Track book. At this point, many questions & answers I see are too advanced for me but I would still like to participate somehow. There is also sometimes an assumption of knowledge which doesn't yet exist when people answer newbie's questions in general support forums. Thanks.

**Comment by fabpot** [comment]
@authentictech Great idea! I've just created a new category for the book: https://github.com/symfony/symfony/discussions/categories/symfony-book-the-fast-track

**Comment by StaffNowa** [comment]
I think we need category where people can share knowledge :)

**Reply by faizanakram99** [reply]
This

**Reply by faizanakram99** [reply]
Also, a general category for discussions. Q/A is not the right category for every discussion.

**Comment by rm5272915-hash** [comment]
Razamuhammad
